src/AI/JakeMan.java

```diff
@@ -0,0 +1,770 @@
+package AI;
+
+import java.util.*;
+import java.util.Map.Entry;
+
+import AI.CommanderProductionInfo;
+import CommandingOfficers.Commander;
+import CommandingOfficers.CommanderAbility;
+import Engine.*;
+import Engine.UnitActionLifecycles.WaitLifecycle;
+import Terrain.*;
+import Units.*;
+
+
+public class JakeMan extends ModularAI
+{
+  private static class instantiator implements AIMaker
+  {
+    @Override
+    public AIController create(Army army)
+    {
+      return new JakeMan(army);
+    }
+
+    @Override
+    public String getName()
+    {
+      return "JakeMan";
+    }
+
+    @Override
+    public String getDescription()
+    {
+      return
+          "Wants your free dudes.\n" +
+          "High-level design by @Lost&Found#6348, ID 824112693123612692.";
+    }
+  }
+  public static final AIMaker info = new instantiator();
+
+  @Override
+  public AIMaker getAIInfo()
+  {
+    return info;
+  }
+
+  // What % base damage I'll ignore when checking safety
+  private static final int    INDIRECT_THREAT_THRESHHOLD = 7;
+  private static final int    DIRECT_THREAT_THRESHHOLD = 13;
```

Comment:
https://grammarist.com/spelling/threshold-or-threshhold/

Not that it's like, important.